manim/mobject/svg/svg_mobject.py

```diff
@@ -234,9 +234,11 @@ def path_string_to_mobject(self, path_string: str, style: dict):
         VMobjectFromSVGPathstring
             A VMobject from the given path string, or d attribute.
         """
-        return SVGPathMobject(
-            path_string, **self.path_string_config, **parse_style(style)
-        )
+        if config.renderer == "opengl":
+            return SVGPathMobject(
+                path_string, **self.path_string_config, **parse_style(style)
+            )
+        return SVGPathMobject(path_string, **parse_style(style))
```

Comment:
I'd prefer for `should_subdivide_sharp_curves` etc. would only be put in `self.path_string_config` if the renderer is opengl, instead of only using `self.path_string_config` if the renderer is opengl.